provider/common/util.go

```diff
@@ -0,0 +1,93 @@
+package common
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+type GraphqlProviderConfig struct {
+	GQLServerUrl string
+	ApiToken     string
+}
+
+func CallExecuteQuery(query string, meta interface{}) (string, error) {
+	url := meta.(*GraphqlProviderConfig).GQLServerUrl
+	api_token := meta.(*GraphqlProviderConfig).ApiToken
+
+	requestBody, err := json.Marshal(map[string]string{
+		"query": query,
+	})
+	if err != nil {
+		return "", err
+	}
+
+	req, err := http.NewRequest("POST", url, bytes.NewBuffer(requestBody))
+	if err != nil {
+		return "", err
+	}
+
+	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("Authorization", api_token)
+
+	client := &http.Client{}
+	resp, err := client.Do(req)
+	if err != nil {
+		return "", err
+	}
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return "", err
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		return "", fmt.Errorf("GraphQL request failed with status: %d - %s", resp.StatusCode, string(body))
+	}
+
+	return string(body), nil
+}
+
+func ReturnQuotedStringList(inputList []interface{}) []string{
+	var envList []string
+    for _, env := range inputList {
```

Comment:
Mix of tabs and spaces isn't good. In golang tabs are preferred for indentation.

Reply:
done https://github.com/Traceableai/terraform-provider-traceable/pull/46/commits/e3af8698bad69897d37dc57f933e31e4041eb91a